app/build.gradle

```diff
@@ -162,14 +163,18 @@ dependencies {
     implementation files('./lib/spotify-app-remote-release-0.7.2.aar')
 
     //Test Setup
+    def mockito_version = '2.7.1'       // Useful for mocking context
```

Comment:
We do not use mockito anywhere. Why are we adding this?

Reply:
I was using it to instantiate my viewmodel because it requires application context